packages/client/src/object/fetchPage.ts

```diff
@@ -129,6 +136,16 @@ export type FetchPageResult<
   >
 >;
 
+export type FetchPageResultWithErrors<
```

Comment:
Should probably rename this type, as its just `FetchPageResult` without the `Promise` wrapper

Reply:
Just realized you said the same thing as me last week!